data-format/message-body.md

```diff
@@ -163,23 +163,23 @@ The Final Frame Sequence number MUST be equal to the total number of frames in t
 
 The initialization vector for the final frame.    
 The IV MUST be a unique IV within the message.   
-The IV length MUST be equal to the IV length of the [algorithm](#algorithm-suite.md) that generated the message. 
-Note: This IV is different from the [Header IV](#message-header.md#iv). 
+The IV length MUST be equal to the IV length of the [algorithm](../framework/algorithm-suites.md) that generated the message. 
+Note: This IV is different from the [Header IV](message-header.md#iv). 
 
 ##### Encrypted Content Length
 
 The length of the encrypted content.  
 
 ##### Encrypted Content
 
-The encrypted data for the final frame, as returned by the [encryption algorithm](#algorithm-suite.md).  
+The encrypted data for the final frame, as returned by the [encryption algorithm](../framework/algorithm-suites.md).  
 
 ##### Authentication Tag
 
 The authentication value for the final frame.    
 It is used to authenticate the final frame.  
 The authentication tag length MUST be equal to the authentication tag length of the algorithm suite 
-specified by the [Algorithm Suite ID](#message-header.md#algorithm-suite-id) field.
+specified by the [Algorithm Suite ID](message-header.md#algorithm-suite-id) field.
 
 ## Test Vectors
 
```

Comment:
The TODO link on line 186 needs to be fixed, it seems to be point to the https://github.com/awslabs/aws-crypto-tools-test-vector-framework.